flamingpy/examples/decoding.py

```diff
@@ -45,6 +45,7 @@ def decode_surface_code(distance, boundaries, ec, noise, draw=True, show=False):
         # Apply noise, measure syndrome, translate to bit values
         CVRHG.apply_noise(cv_noise)
         CVRHG.measure_hom("p", RHG_code.all_syndrome_inds)
+        CVRHG.draw_adjacency(title="CV code adjacency matrix")
```

Comment:
```suggestion
```
No need for this here -- however it looks very cool, so I would include it in the surface code example file